src/objectTranslations/Contact-fr.objectTranslation

```diff
@@ -3494,15 +3510,27 @@
         <name>Preferred_Email__c</name>
         <picklistValues>
             <masterLabel>Alternate</masterLabel>
-            <translation>Autre</translation>
+            <translation>E-mail alternatif</translation>
```

Comment:
Why was the translation changed completely here?

Reply:
I think "Autre" translate to "Other" and the change translates to "Alternate Email". It matches the field label also.

Reply:
Nevermind - I forgot that you were just mapping them to the actual field labels